.pre-commit-config.yaml

```diff
@@ -113,7 +113,7 @@ repos:
           - requirements.txt
           - --header
           - "# generated from manifests external_dependencies"
-  - repo: https://gitlab.com/PyCQA/flake8
+  - repo: https://github.com/PyCQA/flake8
```

Comment:
instead of doing this. You can run pre-commit update in the repo and it will update the config files automatically.

Reply:
Done in https://github.com/OCA/account-fiscal-rule/pull/337.

Reply:
@AaronHForgeFlow @MiquelRForgeFlow thanks I didn't know that